test/components/plots/stackedPlotTests.ts

```diff
@@ -267,6 +267,61 @@ describe("Plots", () => {
     });
   });
 
+  describe("fail safe tests", () => {
+
+    var svg: D3.Selection;
+    var xScale: Plottable.Scale.Category;
+    var yScale: Plottable.Scale.Linear;
+    var data1: Object[];
+    var data2: Object[];
+
+    beforeEach(() => {
+      svg = generateSVG(600, 400);
+
+      data1 = [
+          { x: "A", y: "s", fill: "blue" },
+      ];
+      data2 = [
+          { x: "A", y: 1, fill: "red"},
+      ];
+      xScale = new Plottable.Scale.Category();
+      yScale = new Plottable.Scale.Linear();
+    });
+
+    afterEach(() => {
+      svg.remove();
+    });
+
+    it("conversion fails should be silent in stackedBarPlot", () => {
```

Comment:
It's probably more consistent to use the reverse-dot naming whenever you refer to a `Plottable` object; so "`Plot.StackedBar`" (the actual name), not "`stackedBarPlot`".